sale_rental/views/product.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2014-2016 Akretion (http://www.akretion.com/)
+  @author Alexis de Lattre <alexis.delattre@akretion.com>
+  Copyright 2016 Sodexis
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+
+<record id="product_normal_form_view" model="ir.ui.view">
+    <field name="name">rental.product.product.form</field>
+    <field name="model">product.product</field>
+    <field name="inherit_id" ref="product.product_normal_form_view" />
+    <field name="arch" type="xml">
+        <notebook position="inside">
+            <page name="rental" string="Rental">
+                <group name="rental-service"
+                    attrs="{'invisible': [('type', '!=', 'service')]}">
+                    <field name="rented_product_id"/>
+                </group>
+                <group name="rental-product"
+                    attrs="{'invisible': [('type', '=', 'service')]}"
+                    string="Related Rental Services" col="1">
+                    <field name="rental_service_ids" nolabel="1"/>
+                    <button type="action"
+                        name="%(create_rental_product_action)d"
```

Comment:
@danhvophuong I think here batter to add module name like:  `sale_rental.create_rental_product_action`